src/schema/resolvers/UpdateMutations.ts

```diff
@@ -21,26 +22,38 @@ export class UpdateMutations {
 
   @Mutation(() => UserGroup)
   async addUserToGroup(@Arg('userID') userID: number, @Arg('groupID') groupID: number): Promise<UserGroup> {
-    //console.log(`Adding user (${userID}) to group (${groupID})`);
+    const userService = Container.get<UserService>('UserService');
+
+    //const t0 = performance.now()
     // Try to find the user + groups
-    const user = await User.findOne(userID);
+    const user = await userService.getUser(userID);
+
     if (!user) {
-      const msg = `Unable to find user with ID: ${userID}`;
+      const msg = `Unable to find exactly one user with ID: ${userID}`;
       console.log(msg);
       throw new Error(msg);
     }
 
-    const group = await UserGroup.findOne(groupID);
+    //const t1 = performance.now()
+    const group = await UserGroup.findOne({ where: [ { id: groupID } ] });
+
     if (!group) {
-      const msg = `Unable to find gropu with ID: ${groupID}`;
+      const msg = `Unable to find group with ID: ${groupID}`;
       console.log(msg);
       throw new Error(msg);
     }
+    //const t2 = performance.now()
 
     // Have both user + group so do the add
     group.addUserToGroup(user);
+    //const t3 = performance.now()
     await group.save();
 
+    //const delta1 = t0 - t1;
+    //const delta2 = t2 - t1;
+    //const delta3 = t3 - t2;
+    //const msg = `AddUserToGroup deltas: ${delta1}, ${delta2}, ${delta3}`;
+    //console.log(`AddUserToGroup deltas: ${delta1}, ${delta2}, ${delta3}`);
```

Comment:
Please remove commented code that is not needed or leave a comment why the code is left commented.

Reply:
Commented code is left in there to assist in server-143 issue - there is a performance issue with this mutation